pkgs/by-name/in/inshellisense/package.nix

```diff
@@ -1,17 +1,27 @@
-{ lib, buildNpmPackage, fetchFromGitHub }:
+{ lib, stdenv, buildNpmPackage, fetchFromGitHub, cacert }:
 
 buildNpmPackage rec {
   pname = "inshellisense";
-  version = "0.0.1-rc.4";
+  version = "0.0.1-rc.12";
 
   src = fetchFromGitHub {
     owner = "microsoft";
     repo = pname;
     rev = "refs/tags/${version}";
-    hash = "sha256-PYSonVyclGSH3ArbqJuKrBNGbJaQEp6XemwnHboVwPk=";
+    hash = "sha256-dDjIKVV1dSCIa2Y2d1AQQAw9Rcflh0AnKlwsQSblIhs=";
   };
 
-  npmDepsHash = "sha256-sjr4Hy1/zWPAlVGsMkyQIQcBT86KLaN2/UAaAd7Mn6Q=";
+  npmDepsHash = "sha256-uBsPaUvEiR5oCl8rZvpyNPXSB/Vlcx937lT4WqgekHI=";
+
+  # Needed for dependency `@homebridge/node-pty-prebuilt-multiarch`
+  # On Darwin systems the build fails with,
+  #
+  # npm ERR! ../src/unix/pty.cc:413:13: error: use of undeclared identifier 'openpty'
+  # npm ERR!   int ret = openpty(&master, &slave, nullptr, NULL, static_cast<winsi ze*>(&winp));
+  #
+  # when `node-gyp` tries to build the dep. The below allows `npm` to download the prebuilt binary.
```

Comment:
Does this make the deps hash platform dependent?

Reply:
Assuming you mean `npmDepsHash`? If so, I don't think so. It's building for me on all of `x86_64-darwin` and `aarch64-{darwin,linux}`.